.github/workflows/ci.yml

```diff
@@ -46,8 +46,9 @@ jobs:
           # TODO(cretz): Can upgrade proto when https://github.com/arduino/setup-protoc/issues/99 fixed
           version: "23.x"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-      - run: python -m pip install --upgrade wheel poetry poethepoet
-      - run: poetry install --no-root --all-extras
+      - uses: astral-sh/setup-uv@v5
+      - run: uv tool install poethepoet
```

Comment:
Can we get rid of poe? Does uv offer simple ways to define tasks?

Reply:
Was just commenting on the same thing -- we'll be able to switch away from `poe` when `uv` [includes a task runner](https://github.com/astral-sh/uv/issues/5903), which it sounds to me like will happen in next few months.